src/app/home-body/home-body.component.html

```diff
@@ -6,12 +6,12 @@
           <md-card>
             <md-card-header>
               <img md-card-avatar>
-              <md-card-title>Card with Title</md-card-title>
+              <md-card-title>Lance Johnson</md-card-title>
               <md-card-subtitle>Subtitle</md-card-subtitle>
```

Comment:
Put something cheesy like "Japanese Sensei" in the subtitle. Make sure to capitalize everything.